popup/popup.js

```diff
@@ -288,11 +288,11 @@ var browser = browser || chrome;
 			_backgroundPage.storeParam( 'historylimit', val );
 			this.cleanHistory();
 		}.bind( this ) );
-		wpintegrationWidget.on( 'change', _backgroundPage.storeParam.bind( _backgroundPage, 'wpintegration' ) );
```

Comment:
Replace `.storeParam.bind()` by `.storeParam()`, need to keep an eye on that after merge.

Reply:
what is the difference between storeParam.bind() and storeParam()

Reply:
I inherited this project from someone else, I hack and replicate things, the the `.bind()` may be there for a reason, like to save in the extension's local storage via background.js and not in the page's local storage via popup.js.

It could explain #56 , need to test and keep a eye on it.

Reply:
okay, I will keep an eye on #56.